<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLThreeDViewsControllerWidget</class>
 <widget class="qMRMLWidget" name="qMRMLThreeDViewsControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>92</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>3D Views Controller</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QToolButton" name="ScreenshotButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Capture a screenshot of the full layout, 3D view or slice views.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewCapture.png</normaloff>:/Icons/ViewCapture.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="SceneViewButton">
       <property name="toolTip">
        <string>Capture and name a scene view.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewCamera.png</normaloff>:/Icons/ViewCamera.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="SelectSceneViewMenuButton">
       <property name="toolTip">
        <string>Restore or delete saved scene views.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewCameraSelect.png</normaloff>:/Icons/ViewCameraSelect.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="StackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="NavigationPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="qMRMLNavigationView" name="NavigationView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MagnifyPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="ctkVTKMagnifyView" name="VTKMagnify">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="crosshairColor">
          <color>
           <red>255</red>
           <green>191</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="crosshairType">
          <set>ctkCrosshairLabel::BullsEyeCrosshair</set>
         </property>
         <property name="marginColor">
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="bullsEyeWidth">
          <number>12</number>
         </property>
         <property name="magnification">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkVTKMagnifyView</class>
   <extends>ctkCrosshairLabel</extends>
   <header>ctkVTKMagnifyView.h</header>
  </customwidget>
  <customwidget>
   <class>ctkVTKRenderView</class>
   <extends>QWidget</extends>
   <header>ctkVTKRenderView.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCrosshairLabel</class>
   <extends>QLabel</extends>
   <header>ctkCrosshairLabel.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNavigationView</class>
   <extends>ctkVTKRenderView</extends>
   <header>qMRMLNavigationView.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
